package.json

```diff
@@ -1,71 +1,70 @@
 {
-  "name": "italia-design-react",
-  "version": "1.0.0",
-  "description": "Italia UI Toolkit",
-  "main": "lib/index.js",
+  "name": "design-react-kit",
+  "version": "0.0.2",
+  "description": "Componenti React per Bootstrap 4",
+  "author": "Presidenza del Consiglio dei Ministri",
+  "license": "BSD-3-Clause",
+  "main": "dist/design-react-kit.cjs.js",
+  "module": "dist/design-react-kit.es.js",
+  "engines": {
+    "npm": "^5.0.0"
+  },
   "scripts": {
-    "test": "jest --watch",
-    "test:ci": "jest",
-    "build": "webpack --config webpack.prod.config.js",
-    "lint": "eslint --fix src/",
-    "storybook": "start-storybook -p 8010 -c .storybook",
-    "build-storybook": "build-storybook -c .storybook -o .out",
-    "deploy-storybook": "storybook-to-ghpages"
+    "install-dependencies": "if [ ! -d node_modules ]; then npm install && npx npm-install-peers; fi",
+    "prebuild": "npm run install-dependencies",
+    "build": "rollup -c scripts/rollup.config.js",
+    "prestorybook": "npm run install-dependencies",
+    "storybook": "start-storybook -p 6006",
+    "storybook:build": "build-storybook",
+    "storybook:deploy": "gh-pages -d storybook-static -m 'Update GitHub Pages [ci skip]'",
+    "test": "CI=true react-scripts test --env=jsdom"
```

Comment:
al flag CI=true non ci pensa già il CI "runner" ?

Reply:
Esegue `npm test` non in `watch mode` anche in locale.